### DISCUSSION TITLE
When is the subscribe call back pattern going to be properly removed?

### DISCUSSION THREAD

**Discussion by phillip-whitaker** [ReactiveX/rxjs#6051] [Q&A] [answered]
I know that the previous subscribe pattern is deprecated and you now need to use the subscribe with an observer. But when will we need to move over, do we know in what version it will be removed? date etc.

**Comment by benlesh** [comment, marked as answer]
We're planning on supporting `subscribe(callback: (value: T) => void)` and `subscribe(Partial<Observer<T>>)` past version 8.  only `subscribe(fn, fn)` and `subscribe(fn, fn, fn)` are going to be removed.

**Reply by phillip-whitaker** [reply]
@benlesh Sorry, yes that what I meant - when will the `subscribe(fn, fn)` and `subscribe(fn, fn, fn)`  be removed?
